docs/developer-docs/integrations/icrc-1/icrc1-index-setup.md

```diff
@@ -0,0 +1,245 @@
+                  };
+                };
+                record {
+                  "ts";
+                  variant { Int = 1_695_375_155_855_058_000 : int };
+                };
+              }
+            };
+          };
+        }
+      };
+      variant {
+        Map = vec {
+          record {
+            "phash";
+            variant {
+              Blob = blob "6\f0\02\d0\b0?3x\0b!\18e2\a6)h\b2z\82\cfz\0f\d5\ec>C\e4\05\a3\e0{\e6"
+            };
+          };
+          record { "ts"; variant { Int = 1_695_375_453_656_274_000 : int } };
+          record {
+            "tx";
+            variant {
+              Map = vec {
+                record { "amt"; variant { Int = 100_000 : int } };
+                record { "op"; variant { Text = "mint" } };
+                record {
+                  "to";
+                  variant {
+                    Array = vec {
+                      variant { Blob = blob "e?$\80\12\0e\8d\8f\c0\9f\ae,~\cb\c5\1b\ea\d9\e2\e8\87\8c\d9g\8d\99cf\02" };
+                    }
+                  };
+                };
+              }
+            };
+          };
+        }
+      };
+    };
+    chain_length = 2 : nat64;
+  },
+)
+```
+
+To fetch a vector of all transactions your default account was involved in you can use the following commands.
+Find out the principal of your default account.
+```
+dfx identity get-principal
+```
+In the case of this tutorial this returns
+
+```
+hdq6b-ncywm-yajd5-4inc6-hgpzp-55xnp-py7d5-uqt6o-cv5c6-rrhwa-zqe
+```
+
+Then you can query the transactions for this principal with the default subaccount set by calling. 
+```
+dfx canister call icrc1_index_canister get_account_transactions '(record{account=record {owner = principal "hdq6b-ncywm-yajd5-4inc6-hgpzp-55xnp-py7d5-uqt6o-cv5c6-rrhwa-zqe"}; max_results=2:nat})'
+```
+
+The result will include the intial mint operation as well as the transfer that we made.
+
+```
+(
+  variant {
+    Ok = record {
+      balance = 10_000_000_000 : nat;
+      transactions = vec {
+        record {
+          id = 0 : nat;
+          transaction = record {
+            burn = null;
+            kind = "mint";
+            mint = opt record {
+              to = record {
+                owner = principal "hdq6b-ncywm-yajd5-4inc6-hgpzp-55xnp-py7d5-uqt6o-cv5c6-rrhwa-zqe";
+                subaccount = null;
+              };
+              memo = null;
+              created_at_time = opt (1_695_375_155_855_058_000 : nat64);
+              amount = 10_000_000_000 : nat;
+            };
+            approve = null;
+            timestamp = 1_695_375_155_855_058_000 : nat64;
+            transfer = null;
+          };
+        };
+      };
+      oldest_tx_id = opt (0 : nat);
+    }
+  },
+)
+```
+The ICRC-1 ledger uses AccountIdentifier which are a hashed version of Account (Principal and Subaccount) for privacy. This also means that the returned transactions will show accounts as the hash in bytes rather than the actual Accounts.
```

Comment:
```suggestion
The ICRC-1 ledger uses `AccountIdentifier` which are a hashed version of `Account` (`Principal` and `Subaccount`) for privacy. This also means that the returned transactions will show accounts as the hash in bytes rather than the actual `Accounts`.
```